config.go

```diff
@@ -0,0 +1,98 @@
+package mesosauth
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+const defaultPeriod = 10 * time.Minute
```

Comment:
Should we have a default like this? Should we be using Vault's global TTL defaults rather?